typhos/notes.py

```diff
@@ -99,7 +99,9 @@ def insert_into_yaml(path: Path, device_name: str, data: dict[str, str]) -> None
         with open(path, 'r') as f:
             device_notes = yaml.full_load(f)
     except Exception as ex:
```

Comment:
Hmm, this worries me a bit - because it could nuke your notes in the event of a syntax error.
How about instead:
```suggestion
    except FileNotFoundError:
        # create the file from scratch
    except Exception:
       # report error and do nothing
```